redis/commands/search/commands.py

```diff
@@ -5,12 +5,13 @@
 from redis.client import NEVER_DECODE, Pipeline
 from redis.utils import deprecated_function
 
-from ..helpers import get_protocol_version, parse_to_dict
```

Comment:
It seems that `parse_to_dict` is a dead/unused code now. It should be removed if this is the case with the test case for it.